tests/ui/test_ui_tools.py

```diff
@@ -1976,8 +1975,86 @@ def test_keypress_EDIT_MESSAGE(self, mocker, message_fixture, widget_size,
                 </blockquote>""",
              ("{} {} A\n"
               "{} B\n"
-              "{} {} C"),
-             id="quoted level 2-1-2",
+              "{} \n"
+              "{} {} C\n"),
+             id="quoted level 2-1-2"),
+        case("""<p><a href='https://chat.zulip.org/1'</a>czo</p>
+                <blockquote>
+                    <p><a href='https://chat.zulip.org/2'</a>czo</p>
```

Comment:
This applies to the original links too, but these are actually invalid links - how do these pass?

Reply:
I don't understand what do you mean by "invalid links". Are you pointing out the incorrect syntax used for `a` tag?